### PR TITLE
feat: Change WithMetadata to alias grpc metadata

### PR DESCRIPTION
**Pull request by zzerjae**
Change WithMetadata for alias.

### CODE REVIEW

File: grpc.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ type GRPCLogFormatter interface {
 
 type grpcConfig struct {
 	ignoredMethods map[string]struct{}
-	metadata       map[string]struct{}
+	metadata       map[string]string
```
Comment (dlsrb6342) [inline grpc.go:49]: > colon is used for dividing header name and alias key. If colon is not exist, use header name as a key. 

I suggested colon separated string for alias. do you think alias map is better?

Comment (zzerjae) [reply grpc.go:49]: I was mistaken. It's corrected.

File: grpc_option.go [hunk 2]
```diff
@@ -15,14 +17,30 @@ func WithIgnoredMethods(ms ...string) grpcOption {
 }
 
 // WithMetadata specifies headers to be captured by the logger.
+// The key of the ms is the name of the metadata.
+// And the value is the name to set in the log field.
+// If the value is omitted, the name of the metadata is used as it is.
```
Comment (dlsrb6342) [inline grpc_option.go:22]: need to update docs

Comment (zzerjae) [reply grpc_option.go]: thx, fixed

**Review by dlsrb6342 — APPROVED** [review]

**Review by dlsrb6342 — APPROVED** [review]

**Pull request merged**